package/scie-pants.lift.json

```diff
@@ -118,6 +118,10 @@
               "{scie.bindings.configure:PANTS_SHA_FIND_LINKS}"
             ]
           },
+          "pants-dev": {
```

Comment:
So a lift entry means:
1. scie-jump -> re-exec scie-pants Rust
2. scie-pants Rust -> re-exec scie-jump
3. scie-jump -> re-exec pants-dev

Doing the re-direct in scie-pants saves #3 which is likely ~300us. So, you could get away with it, but this is not a style choice or even a perf choice really. Those other cases all need binding commands to run and that's why they re-direct. You do not, you know what to re-exec and it needs no bindings setup at all.

Reply:
Here's a rule of thumb - perhaps even a hard rule, I can't see exceptions yet:

If you define a named "commands" entry that uses 0 placeholders; that's incorrect unless your scie is a BusyBox (it has no default "" command).

Reply:
So, you did use a placeholder here ... but you had to fill that in yourself in the default command; thus the wasteful bit to use an extra re-exec than needed for dispatch.

Reply:
Makes total sense, thank you.

Spent 80% of the time spent on this PR on reading code to build my mental map of the terrain here (still incomplete ;)